packages/kbn-dependency-usage/README.md

```diff
@@ -0,0 +1,124 @@
+# @kbn/dependency-usage
```

Comment:
One of the use cases from https://github.com/elastic/kibana/issues/196767 isn't implemented yet: 

> ## Script to identify dependency ownership
> 
> A script should exist which determines dependency ownership.
> A dependency is Owned by a team if a corresponding entry exists in `renovate.json` with a `reviewers` property assigned to a `team:*` value.
> In this example, the `@elastic/charts` dependency is owned by the `@elastic/visualizations` team.
> ```json
> {
> 	"groupName": "@elastic/charts",
> 	"matchDepNames": ["@elastic/charts"],
> 	"reviewers": ["team:visualizations", "markov00", "nickofthyme"],
> 	"matchBaseBranches": ["main"],
> 	"labels": ["release_note:skip", "backport:skip", "Team:Visualizations"],
> 	"enabled": true
> },
> ```
> 
> We should be able to answer the following questions:
> 1/ Who owns the `react` dependency?
> 2/ Which dependencies do not have an owner?
> 3/ Which dependencies are owned by the `@elastic/kibana-security` team?
> 4/ Generate a full dependency ownership report.

I am happy for us to address this in a follow-up PR, as it's not 100% related to the rest of the operations. I'm only mentioning this for completeness, as this PR is currently set to close #196767 when it merges.

Reply:
yeah, this one is on my list, didn't include it here